pkg/databackup/value.go

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright <year> The <project-name> Author.
```

Comment:
The first line of the content should be `Copyright 2021 The Fluid Author.`

Reply:
I accidentally deleted the previous branch, I have re-opened another pr now, please check it